tests/jballerina-unit-test/src/test/resources/test-src/expressions/unaryoperations/unary-operation.bal

```diff
@@ -106,49 +106,49 @@ function testComplementOperator() {
 function testUnaryOperationsWithIntSubtypes() {
     int:Unsigned8 x1 = 7;
     int y1 = ~x1;
-    int:Unsigned8 x2 = +7;
```

Comment:
Why are we changing these? I think we need tests like these also.

Shall we leave these as is and add a new set of tests?

Reply:
will add separate tests

Reply:
Shall we undo these changes?